source/windows/iocp/socket.c

```diff
@@ -2203,6 +2203,18 @@ int aws_socket_set_options(struct aws_socket *socket, const struct aws_socket_op
                 &bytes_returned,
                 NULL,
                 NULL);
+
+#ifdef TCP_KEEPCNT
```

Comment:
When is this not defined on windows?

Reply:
It's available Windows 10 1703 and later